tsl/src/continuous_aggs/create.c

```diff
@@ -1155,14 +1162,17 @@ cagg_validate_query(const Query *query, const bool finalized, const char *cagg_s
 	CAggTimebucketInfo bucket_info, bucket_info_parent;
 	Cache *hcache;
 	Hypertable *ht = NULL, *ht_parent = NULL;
-	RangeTblRef *rtref = NULL;
-	RangeTblEntry *rte;
-	List *fromList;
+	RangeTblRef *rtref = NULL, *rtref2 = NULL;
+	RangeTblEntry *rte = NULL, *rte2 = NULL;
```

Comment:
Wondering if we could use better names for those variables, maybe something like `rteLeft` and `rteRight` (and the same for `RangeTblRef`)??

Reply:
> Need to add some more tests:
> 
> * caggs on caggs with join
This is already added in cagg_joins test case